app/components/markdown/hashtag/index.tsx

```diff
@@ -17,7 +17,7 @@ const Hashtag = ({hashtag, linkStyle}: HashtagProps) => {
         await dismissAllModals();
         await popToRoot();
 
-        showSearchModal('#' + hashtag);
+        // showSearchModal('#' + hashtag);
```

Comment:
Any reason to keep this as a comment here?

Reply:
Search is not ready and I don't want us to miss it